src/components/nabla/common/TransactionProgress.tsx

```diff
@@ -26,43 +28,61 @@ const getErrorMessage = (data?: ExecuteMessageResult['result']) => {
   }
 };
 
-export function TransactionProgress({ mutation, children, onClose }: TransactionProgressProps): JSX.Element | null {
-  if (mutation.isIdle) return null;
+function getExplorerUrl(tenant: TenantName, data?: ExecuteMessageResult['execution']) {
+  if (tenant === TenantName.Pendulum && data?.type === 'extrinsic') {
+    return `https://pendulum.subscan.io/extrinsic/${data.txHash.toHex()}`;
+  }
+}
+
+export function TransactionProgress({ mutation, children, onClose }: TransactionProgressProps) {
+  const { tenantName } = useGlobalState();
   const errorMsg = getErrorMessage(mutation.data?.result);
+
+  const explorerUrl = getExplorerUrl(tenantName, mutation.data?.execution);
+
+  if (mutation.isIdle) return null;
+
   if (mutation.isLoading) {
     const isPending = false; // TODO: currently there is not status for this (waiting confirmation in wallet)
     return (
       <>
-        <div className="flex flex-col items-center justify-center text-center mt-4">
+        <div className="mt-4 flex flex-col items-center justify-center text-center">
           <Spinner size={100} color="#ddd" />
-          <h4 className="text-2xl mt-12 text-[--text]">
+          <h4 className="mt-12 text-2xl text-[--text]">
             {isPending ? 'Waiting for confirmation' : 'Executing transaction'}
           </h4>
           {children}
-          <p className="text-neutral-500 my-5">
+          <p className="my-5 text-neutral-500">
             {isPending ? 'Confirm this transaction in your wallet' : 'Proceed in your wallet'}
           </p>
         </div>
       </>
     );
   }
+
   return (
     <>
       <div className="center mt-6">
         {mutation.isSuccess ? (
-          <CheckCircleIcon className="w-36 h-36 text-green-400" stroke-width={1} />
+          <CheckCircleIcon className="h-36 w-36 text-green-400" stroke-width={1} />
         ) : (
-          <ExclamationCircleIcon className="w-36 h-36 text-red-400" stroke-width={1} />
+          <ExclamationCircleIcon className="h-36 w-36 text-red-400" stroke-width={1} />
         )}
       </div>
-      <div className="text-center mt-4">
+      <div className="mt-4 text-center">
         <h4 className="text-2xl text-[--text]">
           {mutation.isSuccess ? 'Transaction successful' : 'Transaction failed'}
         </h4>
       </div>
-      {!mutation.isSuccess && !!errorMsg && <p className="text-center mt-1">{errorMsg}</p>}
+      {!mutation.isSuccess && !!errorMsg && <p className="mt-1 text-center">{errorMsg}</p>}
+      <div className="mt-6"></div>
+      {!!explorerUrl && (
```

Comment:
Just a question because it is the first time I see it that way: Is there a difference between `!!explorerUrl && ...` and `explorerUrl &&` since `explorerUrl` can only be `undefined` or a truthy (nonempty) string? 

[From the official docs](https://react.dev/learn/conditional-rendering#logical-and-operator-):
> React considers `false` as a “hole” in the JSX tree, just like `null` or `undefined`, and doesn’t render anything in its place.

Reply:
No you are right, there is no need for this here. I just used it to cast it to a boolean value but that's not really necessary here. Removed.